### PR TITLE
Fix remaining explicit URLs (incl. filter tabs)

### PR DESCRIPTION
**Pull request by rmloveland**
Addresses DOC-10953

Summary of changes:

- Convert ALL remaining `www.cockroachlabs.com/docs/...` links to use Liquid template syntax across all currently supported versions of CockroachDB (v23.1, v23.2, v24.1, v24.2).

- NB. *Only* links were edited, no content changes were made whatsoever

### CODE REVIEW

**Comment by rmloveland** [comment]
cc @mdlinville FYI, this should be the last PR needing a port to `cloud-2.0` branch.  The changes needed were too extensive to be done via script, so they'll have to be merged.  The good news is this PR _only_ touches links, there are no content changes, so it "should merge properly" (tm) (famous last words lol).

File: src/current/v23.2/transaction-retry-error-reference.md [hunk 1]
```diff
@@ -128,7 +128,7 @@ TransactionRetryWithProtoRefreshError: ... RETRY_SERIALIZABLE ...
 ```
 restart transaction: TransactionRetryWithProtoRefreshError: TransactionRetryError: retry txn (RETRY_SERIALIZABLE  - failed preemptive refresh due to conflicting locks on /Table/106/1/918951292305080321/0 [reason=wait_policy] - conflicting txn: meta={id=1b2bf263 key=/Table/106/1/918951292305080321/0 iso=Serializable pri=0.00065863 epo=0 ts=1700512205.521833000,2 min=1700512148.761403000,0 seq=1}): "sql txn" meta={id=07d42834 key=/Table/106/1/918951292305211393/0 iso=Serializable pri=0.01253025 epo=0 ts=1700512229.378453000,2 min=1700512130.342117000,0 seq=2} lock=true stat=PENDING rts=1700512130.342117000,0 wto=false gul=1700512130.842117000,0
 SQLSTATE: 40001
-HINT: See: https://www.cockroachlabs.com/docs/v23.2/transaction-retry-error-reference.html#retry_serializable
+HINT: See: {% link {{ page.version.version }}/transaction-retry-error-reference.md %}#retry_serializable
```
Comment (taroface) [inline src/current/v23.2/transaction-retry-error-reference.md:131]: Ditto

File: src/current/v23.2/upgrade-cockroachdb-kubernetes.md [hunk 2]
```diff
@@ -32,11 +32,11 @@ If you [deployed CockroachDB on Red Hat OpenShift]({% link {{ page.version.versi
...
+    To upgrade to a new major version, you must first be on a production release of the previous version. The release does not need to be the latest production release of the previous version, but it must be a production [release]({% link releases/index.md %}) and not a testing release (alpha/beta).
 
     Therefore, in order to upgrade to {{ page.version.version }}, you must be on a production release of {{ previous_version }}.
 
-    1. If you are upgrading to {{ page.version.version }} from a production release earlier than {{ previous_version }}, or from a testing release (alpha/beta), first [upgrade to a production release of {{ previous_version }}](https://www.cockroachlabs.com/docs/v21.1/operate-cockroachdb-kubernetes#upgrade-the-cluster). Be sure to complete all the steps.
+    1. If you are upgrading to {{ page.version.version }} from a production release earlier than {{ previous_version }}, or from a testing release (alpha/beta), first [upgrade to a production release of {{ previous_version }}]({% link v21.1/operate-cockroachdb-kubernetes.md %}#upgrade-the-cluster). Be sure to complete all the steps.
```
Comment (taroface) [inline src/current/v23.2/upgrade-cockroachdb-kubernetes.md:39]: Same comment as earlier

```suggestion
    1. If you are upgrading to {{ page.version.version }} from a production release earlier than {{ previous_version }}, or from a testing release (alpha/beta), first [upgrade to a production release of {{ previous_version }}]({% link {{ previous_version }}/operate-cockroachdb-kubernetes.md %}#upgrade-the-cluster). Be sure to complete all the steps.
```

File: src/current/v24.1/column-level-encryption.md [hunk 3]
```diff
@@ -50,7 +50,7 @@ For usage examples, see:
 These functions require an [Enterprise license]({% link {{ page.version.version }}/enterprise-licensing.md %}). If you do not have an Enterprise license, you will see an error message like the following if you try to use them:
 
 ~~~
-ERROR: decrypt(): use of this cryptographic function (https://www.cockroachlabs.com/docs/stable/functions-and-operators#cryptographic-functions) requires an enterprise license. see https://cockroachlabs.com/pricing for details on how to enable enterprise features
+ERROR: decrypt(): use of this cryptographic function ({% link {{ page.version.version }}/functions-and-operators.md %}#cryptographic-functions) requires an enterprise license. see https://cockroachlabs.com/pricing for details on how to enable enterprise features
```
Comment (taroface) [inline src/current/v24.1/column-level-encryption.md:53]: Ditto

File: src/current/v24.1/demo-serializable.md [hunk 4]
```diff
@@ -456,7 +456,7 @@ When you repeat the scenario on CockroachDB, you'll see that the anomaly is prev
     ~~~
     ERROR: restart transaction: TransactionRetryWithProtoRefreshError: TransactionRetryError: retry txn (RETRY_SERIALIZABLE - failed preemptive refresh due to encountered recently written committed value /Table/105/1/20001/1/0 @1700513356.063385000,2): "sql txn" meta={id=10f4abbc key=/Table/105/1/20001/2/0 iso=Serializable pri=0.00167708 epo=0 ts=1700513366.194063000,2 min=1700513327.262632000,0 seq=1} lock=true stat=PENDING rts=1700513327.262632000,0 wto=false gul=1700513327.762632000,0
     SQLSTATE: 40001
-    HINT: See: https://www.cockroachlabs.com/docs/v23.2/transaction-retry-error-reference.html#retry_serializable
+    HINT: See: {% link {{ page.version.version }}/transaction-retry-error-reference.md %}#retry_serializable
```
Comment (taroface) [inline src/current/v24.1/demo-serializable.md:459]: Ditto

File: src/current/v24.1/start-a-local-cluster.md [hunk 5]
```diff
@@ -57,7 +57,7 @@ This section shows how to start a cluster interactively. In production, operator
     * - Any user, connecting as root, can read or write any data in your cluster.
     * - There is no network encryption nor authentication, and thus no confidentiality.
     *
-    * Check out how to secure your cluster: https://www.cockroachlabs.com/docs/{{ page.version.version }}/secure-a-cluster.html
+    * Check out how to secure your cluster: {% link {{ page.version.version }}/secure-a-cluster.md %}
```
Comment (taroface) [inline src/current/v24.1/start-a-local-cluster.md:60]: Ditto

File: src/current/v23.2/start-a-local-cluster.md [hunk 6]
```diff
@@ -57,7 +57,7 @@ This section shows how to start a cluster interactively. In production, operator
     * - Any user, connecting as root, can read or write any data in your cluster.
     * - There is no network encryption nor authentication, and thus no confidentiality.
     *
-    * Check out how to secure your cluster: https://www.cockroachlabs.com/docs/{{ page.version.version }}/secure-a-cluster.html
+    * Check out how to secure your cluster: {% link {{ page.version.version }}/secure-a-cluster.md %}
```
Comment (taroface) [inline src/current/v23.2/start-a-local-cluster.md:60]: Ditto

File: src/current/v24.2/start-a-local-cluster.md [hunk 7]
```diff
@@ -57,7 +57,7 @@ This section shows how to start a cluster interactively. In production, operator
     * - Any user, connecting as root, can read or write any data in your cluster.
     * - There is no network encryption nor authentication, and thus no confidentiality.
     *
-    * Check out how to secure your cluster: https://www.cockroachlabs.com/docs/{{ page.version.version }}/secure-a-cluster.html
+    * Check out how to secure your cluster: {% link {{ page.version.version }}/secure-a-cluster.md %}
```
Comment (taroface) [inline src/current/v24.2/start-a-local-cluster.md:60]: Ditto

File: src/current/v24.1/transaction-retry-error-reference.md [hunk 8]
```diff
@@ -135,7 +135,7 @@ The error message for `RETRY_SERIALIZABLE` contains additional information about
 ```
 restart transaction: TransactionRetryWithProtoRefreshError: TransactionRetryError: retry txn (RETRY_SERIALIZABLE  - failed preemptive refresh due to conflicting locks on /Table/106/1/918951292305080321/0 [reason=wait_policy] - conflicting txn: meta={id=1b2bf263 key=/Table/106/1/918951292305080321/0 iso=Serializable pri=0.00065863 epo=0 ts=1700512205.521833000,2 min=1700512148.761403000,0 seq=1}): "sql txn" meta={id=07d42834 key=/Table/106/1/918951292305211393/0 iso=Serializable pri=0.01253025 epo=0 ts=1700512229.378453000,2 min=1700512130.342117000,0 seq=2} lock=true stat=PENDING rts=1700512130.342117000,0 wto=false gul=1700512130.842117000,0
 SQLSTATE: 40001
-HINT: See: https://www.cockroachlabs.com/docs/v23.2/transaction-retry-error-reference.html#retry_serializable
+HINT: See: {% link {{ page.version.version }}/transaction-retry-error-reference.md %}#retry_serializable
```
Comment (taroface) [inline src/current/v24.1/transaction-retry-error-reference.md:138]: Ditto

File: src/current/v24.1/upgrade-cockroachdb-kubernetes.md [hunk 9]
```diff
@@ -32,11 +32,11 @@ If you [deployed CockroachDB on Red Hat OpenShift]({% link {{ page.version.versi
...
+    To upgrade to a new major version, you must first be on a production release of the previous version. The release does not need to be the latest production release of the previous version, but it must be a production [release]({% link releases/index.md %}) and not a testing release (alpha/beta).
 
     Therefore, in order to upgrade to {{ page.version.version }}, you must be on a production release of {{ previous_version }}.
 
-    1. If you are upgrading to {{ page.version.version }} from a production release earlier than {{ previous_version }}, or from a testing release (alpha/beta), first [upgrade to a production release of {{ previous_version }}](https://www.cockroachlabs.com/docs/v21.1/operate-cockroachdb-kubernetes#upgrade-the-cluster). Be sure to complete all the steps.
+    1. If you are upgrading to {{ page.version.version }} from a production release earlier than {{ previous_version }}, or from a testing release (alpha/beta), first [upgrade to a production release of {{ previous_version }}]({% link v21.1/operate-cockroachdb-kubernetes.md %}#upgrade-the-cluster). Be sure to complete all the steps.
```
Comment (taroface) [inline src/current/v24.1/upgrade-cockroachdb-kubernetes.md:39]: Same comment as earlier.

```suggestion
    1. If you are upgrading to {{ page.version.version }} from a production release earlier than {{ previous_version }}, or from a testing release (alpha/beta), first [upgrade to a production release of {{ previous_version }}]({% link {{ previous_version }}/operate-cockroachdb-kubernetes.md %}#upgrade-the-cluster). Be sure to complete all the steps.
```

File: src/current/v24.2/column-level-encryption.md [hunk 10]
```diff
@@ -31,7 +31,7 @@ For usage examples, see:
 These functions require an [Enterprise license]({% link {{ page.version.version }}/enterprise-licensing.md %}). If you do not have an Enterprise license, you will see an error message like the following if you try to use them:
 
 ~~~
-ERROR: encrypt(): use of this cryptographic function (https://www.cockroachlabs.com/docs/stable/functions-and-operators#cryptographic-functions) requires an enterprise license. see https://cockroachlabs.com/pricing for details on how to enable enterprise features
+ERROR: encrypt(): use of this cryptographic function ({% link {{ page.version.version }}/functions-and-operators.md %}#cryptographic-functions) requires an enterprise license. see https://cockroachlabs.com/pricing for details on how to enable enterprise features
```
Comment (taroface) [inline src/current/v24.2/column-level-encryption.md:34]: Ditto

File: src/current/v24.2/column-level-encryption.md [hunk 11]
```diff
@@ -50,7 +50,7 @@ For usage examples, see:
 These functions require an [Enterprise license]({% link {{ page.version.version }}/enterprise-licensing.md %}). If you do not have an Enterprise license, you will see an error message like the following if you try to use them:
 
 ~~~
-ERROR: decrypt(): use of this cryptographic function (https://www.cockroachlabs.com/docs/stable/functions-and-operators#cryptographic-functions) requires an enterprise license. see https://cockroachlabs.com/pricing for details on how to enable enterprise features
+ERROR: decrypt(): use of this cryptographic function ({% link {{ page.version.version }}/functions-and-operators.md %}#cryptographic-functions) requires an enterprise license. see https://cockroachlabs.com/pricing for details on how to enable enterprise features
```
Comment (taroface) [inline src/current/v24.2/column-level-encryption.md:53]: Ditto

File: src/current/v24.2/demo-serializable.md [hunk 12]
```diff
@@ -456,7 +456,7 @@ When you repeat the scenario on CockroachDB, you'll see that the anomaly is prev
     ~~~
     ERROR: restart transaction: TransactionRetryWithProtoRefreshError: TransactionRetryError: retry txn (RETRY_SERIALIZABLE - failed preemptive refresh due to encountered recently written committed value /Table/105/1/20001/1/0 @1700513356.063385000,2): "sql txn" meta={id=10f4abbc key=/Table/105/1/20001/2/0 iso=Serializable pri=0.00167708 epo=0 ts=1700513366.194063000,2 min=1700513327.262632000,0 seq=1} lock=true stat=PENDING rts=1700513327.262632000,0 wto=false gul=1700513327.762632000,0
     SQLSTATE: 40001
-    HINT: See: https://www.cockroachlabs.com/docs/v23.2/transaction-retry-error-reference.html#retry_serializable
+    HINT: See: {% link {{ page.version.version }}/transaction-retry-error-reference.md %}#retry_serializable
```
Comment (taroface) [inline src/current/v24.2/demo-serializable.md:459]: Ditto

File: src/current/v24.2/transaction-retry-error-reference.md [hunk 13]
```diff
@@ -135,7 +135,7 @@ The error message for `RETRY_SERIALIZABLE` contains additional information about
 ```
 restart transaction: TransactionRetryWithProtoRefreshError: TransactionRetryError: retry txn (RETRY_SERIALIZABLE  - failed preemptive refresh due to conflicting locks on /Table/106/1/918951292305080321/0 [reason=wait_policy] - conflicting txn: meta={id=1b2bf263 key=/Table/106/1/918951292305080321/0 iso=Serializable pri=0.00065863 epo=0 ts=1700512205.521833000,2 min=1700512148.761403000,0 seq=1}): "sql txn" meta={id=07d42834 key=/Table/106/1/918951292305211393/0 iso=Serializable pri=0.01253025 epo=0 ts=1700512229.378453000,2 min=1700512130.342117000,0 seq=2} lock=true stat=PENDING rts=1700512130.342117000,0 wto=false gul=1700512130.842117000,0
 SQLSTATE: 40001
-HINT: See: https://www.cockroachlabs.com/docs/v23.2/transaction-retry-error-reference.html#retry_serializable
+HINT: See: {% link {{ page.version.version }}/transaction-retry-error-reference.md %}#retry_serializable
```
Comment (taroface) [inline src/current/v24.2/transaction-retry-error-reference.md:138]: Ditto

File: src/current/v24.2/upgrade-cockroachdb-kubernetes.md [hunk 14]
```diff
@@ -32,11 +32,11 @@ If you [deployed CockroachDB on Red Hat OpenShift]({% link {{ page.version.versi
...
+    To upgrade to a new major version, you must first be on a production release of the previous version. The release does not need to be the latest production release of the previous version, but it must be a production [release]({% link releases/index.md %}) and not a testing release (alpha/beta).
 
     Therefore, in order to upgrade to {{ page.version.version }}, you must be on a production release of {{ previous_version }}.
 
-    1. If you are upgrading to {{ page.version.version }} from a production release earlier than {{ previous_version }}, or from a testing release (alpha/beta), first [upgrade to a production release of {{ previous_version }}](https://www.cockroachlabs.com/docs/v21.1/operate-cockroachdb-kubernetes#upgrade-the-cluster). Be sure to complete all the steps.
+    1. If you are upgrading to {{ page.version.version }} from a production release earlier than {{ previous_version }}, or from a testing release (alpha/beta), first [upgrade to a production release of {{ previous_version }}]({% link v21.1/operate-cockroachdb-kubernetes.md %}#upgrade-the-cluster). Be sure to complete all the steps.
```
Comment (taroface) [inline src/current/v24.2/upgrade-cockroachdb-kubernetes.md:39]: Same comment as earlier.

```suggestion
    1. If you are upgrading to {{ page.version.version }} from a production release earlier than {{ previous_version }}, or from a testing release (alpha/beta), first [upgrade to a production release of {{ previous_version }}]({% link {{ previous_version }}/operate-cockroachdb-kubernetes.md %}#upgrade-the-cluster). Be sure to complete all the steps.
```

**Review by taroface — COMMENTED** [review]
Found a few small things, mainly: 1) URLs in CLI output shouldn't be changed; 2) one of the links in the K8s upgrade doc was hard-coded (this is on me), so I'm suggesting an opportunistic fix here.

File: src/current/_includes/v23.1/backward-incompatible/alpha.1.md
[the remark this answers is not in the document]
Comment (rmloveland) [reply src/current/_includes/v23.1/backward-incompatible/alpha.1.md:1]: oh that's a mistake, that was meant to be a temporary local test, I'll add it back now

File: src/current/v23.1/geoserver.md [hunk 15]
```diff
@@ -117,7 +117,7 @@ In the left-hand navigation menu, click **Data > Workspaces**. The **Workspaces*
 On the **New Workspace** page, enter the following information:
 
 - In the **Name** field, enter the text "spatial-tutorial".
-- In the **Namespace URI** field, enter the URL for the spatial tutorial where this data set is used: https://www.cockroachlabs.com/docs/stable/spatial-data.html.
+- In the **Namespace URI** field, enter the URL for the spatial tutorial where this data set is used: `{% link {{ page.version.version }}/spatial-tutorial.md %}`.
```
[the remark this answers is not in the document]
Comment (rmloveland) [reply src/current/v23.1/geoserver.md:120]: thanks!

File: src/current/v23.1/start-a-local-cluster.md [hunk 16]
```diff
@@ -57,7 +57,7 @@ This section shows how to start a cluster interactively. In production, operator
     * - Any user, connecting as root, can read or write any data in your cluster.
     * - There is no network encryption nor authentication, and thus no confidentiality.
     *
-    * Check out how to secure your cluster: https://www.cockroachlabs.com/docs/{{ page.version.version }}/secure-a-cluster.html
```
[the remark this answers is not in the document]
Comment (rmloveland) [reply src/current/v23.1/start-a-local-cluster.md:60]: You're right, changed it back - thanks

File: src/current/v23.1/upgrade-cockroachdb-kubernetes.md [hunk 17]
```diff
@@ -32,11 +32,11 @@ If you [deployed CockroachDB on Red Hat OpenShift]({% link {{ page.version.versi
...
+    To upgrade to a new major version, you must first be on a production release of the previous version. The release does not need to be the latest production release of the previous version, but it must be a production [release]({% link releases/index.md %}) and not a testing release (alpha/beta).
 
     Therefore, in order to upgrade to {{ page.version.version }}, you must be on a production release of {{ previous_version }}.
 
-    1. If you are upgrading to {{ page.version.version }} from a production release earlier than {{ previous_version }}, or from a testing release (alpha/beta), first [upgrade to a production release of {{ previous_version }}](https://www.cockroachlabs.com/docs/v21.1/operate-cockroachdb-kubernetes#upgrade-the-cluster). Be sure to complete all the steps.
+    1. If you are upgrading to {{ page.version.version }} from a production release earlier than {{ previous_version }}, or from a testing release (alpha/beta), first [upgrade to a production release of {{ previous_version }}]({% link v21.1/operate-cockroachdb-kubernetes.md %}#upgrade-the-cluster). Be sure to complete all the steps.
```
[the remark this answers is not in the document]
Comment (rmloveland) [reply src/current/v23.1/upgrade-cockroachdb-kubernetes.md:39]: updated, thanks!

File: src/current/v23.2/column-level-encryption.md [hunk 18]
```diff
@@ -31,7 +31,7 @@ For usage examples, see:
 These functions require an [Enterprise license]({% link {{ page.version.version }}/enterprise-licensing.md %}). If you do not have an Enterprise license, you will see an error message like the following if you try to use them:
 
 ~~~
-ERROR: encrypt(): use of this cryptographic function (https://www.cockroachlabs.com/docs/stable/functions-and-operators#cryptographic-functions) requires an enterprise license. see https://cockroachlabs.com/pricing for details on how to enable enterprise features
+ERROR: encrypt(): use of this cryptographic function ({% link {{ page.version.version }}/functions-and-operators.md %}#cryptographic-functions) requires an enterprise license. see https://cockroachlabs.com/pricing for details on how to enable enterprise features
```
[the remark this answers is not in the document]
Comment (rmloveland) [reply src/current/v23.2/column-level-encryption.md:34]: Fixed!

File: src/current/v23.2/column-level-encryption.md [hunk 19]
```diff
@@ -50,7 +50,7 @@ For usage examples, see:
 These functions require an [Enterprise license]({% link {{ page.version.version }}/enterprise-licensing.md %}). If you do not have an Enterprise license, you will see an error message like the following if you try to use them:
 
 ~~~
-ERROR: decrypt(): use of this cryptographic function (https://www.cockroachlabs.com/docs/stable/functions-and-operators#cryptographic-functions) requires an enterprise license. see https://cockroachlabs.com/pricing for details on how to enable enterprise features
+ERROR: decrypt(): use of this cryptographic function ({% link {{ page.version.version }}/functions-and-operators.md %}#cryptographic-functions) requires an enterprise license. see https://cockroachlabs.com/pricing for details on how to enable enterprise features
```
[the remark this answers is not in the document]
Comment (rmloveland) [reply src/current/v23.2/column-level-encryption.md:53]: Fixed!

File: src/current/v23.2/demo-serializable.md [hunk 20]
```diff
@@ -456,7 +456,7 @@ When you repeat the scenario on CockroachDB, you'll see that the anomaly is prev
     ~~~
     ERROR: restart transaction: TransactionRetryWithProtoRefreshError: TransactionRetryError: retry txn (RETRY_SERIALIZABLE - failed preemptive refresh due to encountered recently written committed value /Table/105/1/20001/1/0 @1700513356.063385000,2): "sql txn" meta={id=10f4abbc key=/Table/105/1/20001/2/0 iso=Serializable pri=0.00167708 epo=0 ts=1700513366.194063000,2 min=1700513327.262632000,0 seq=1} lock=true stat=PENDING rts=1700513327.262632000,0 wto=false gul=1700513327.762632000,0
     SQLSTATE: 40001
-    HINT: See: https://www.cockroachlabs.com/docs/v23.2/transaction-retry-error-reference.html#retry_serializable
+    HINT: See: {% link {{ page.version.version }}/transaction-retry-error-reference.md %}#retry_serializable
```
[the remark this answers is not in the document]
Comment (rmloveland) [reply src/current/v23.2/demo-serializable.md:459]: Fixed, thanks!  Totes agreed about that. I think i got into a fugue state at some point during this task :-)

File: src/current/v23.2/transaction-retry-error-reference.md (hunk 1 above)
Comment (rmloveland) [reply src/current/v23.2/transaction-retry-error-reference.md:131]: Fixed!

File: src/current/v23.2/upgrade-cockroachdb-kubernetes.md (hunk 2 above)
Comment (rmloveland) [reply src/current/v23.2/upgrade-cockroachdb-kubernetes.md:39]: Fixed

File: src/current/v24.1/column-level-encryption.md (hunk 3 above)
Comment (rmloveland) [reply src/current/v24.1/column-level-encryption.md:53]: Fixed

File: src/current/v24.1/demo-serializable.md (hunk 4 above)
Comment (rmloveland) [reply src/current/v24.1/demo-serializable.md:459]: Fixed

File: src/current/v24.1/start-a-local-cluster.md (hunk 5 above)
Comment (rmloveland) [reply src/current/v24.1/start-a-local-cluster.md:60]: Fixed

File: src/current/v23.2/start-a-local-cluster.md (hunk 6 above)
Comment (rmloveland) [reply src/current/v23.2/start-a-local-cluster.md:60]: Fixed

File: src/current/v24.1/transaction-retry-error-reference.md (hunk 8 above)
Comment (rmloveland) [reply src/current/v24.1/transaction-retry-error-reference.md:138]: Fixed

File: src/current/v24.1/upgrade-cockroachdb-kubernetes.md (hunk 9 above)
Comment (rmloveland) [reply src/current/v24.1/upgrade-cockroachdb-kubernetes.md:39]: Fixed, thanks!

File: src/current/v24.2/column-level-encryption.md (hunk 10 above)
Comment (rmloveland) [reply src/current/v24.2/column-level-encryption.md:34]: Fixed

File: src/current/v24.2/column-level-encryption.md (hunk 11 above)
Comment (rmloveland) [reply src/current/v24.2/column-level-encryption.md:53]: Fixed

File: src/current/v24.2/demo-serializable.md (hunk 12 above)
Comment (rmloveland) [reply src/current/v24.2/demo-serializable.md:459]: Fixed

File: src/current/v24.2/transaction-retry-error-reference.md (hunk 13 above)
Comment (rmloveland) [reply src/current/v24.2/transaction-retry-error-reference.md:138]: Fixed

File: src/current/v24.2/upgrade-cockroachdb-kubernetes.md (hunk 14 above)
Comment (rmloveland) [reply src/current/v24.2/upgrade-cockroachdb-kubernetes.md:39]: Fixed, thanks!

**Comment by rmloveland** [comment]
Thanks for the review @taroface - just pushed commits that hopefully address all of your feedback!

Once build is green PTAL!

Thanks for reviewing this!

File: src/current/v23.1/upgrade-cockroachdb-kubernetes.md [hunk 21]
```diff
@@ -32,11 +32,11 @@ If you [deployed CockroachDB on Red Hat OpenShift]({% link {{ page.version.versi
...
+    To upgrade to a new major version, you must first be on a production release of the previous version. The release does not need to be the latest production release of the previous version, but it must be a production [release]({% link releases/index.md %}) and not a testing release (alpha/beta).
 
     Therefore, in order to upgrade to {{ page.version.version }}, you must be on a production release of {{ previous_version }}.
 
-    1. If you are upgrading to {{ page.version.version }} from a production release earlier than {{ previous_version }}, or from a testing release (alpha/beta), first [upgrade to a production release of {{ previous_version }}](https://www.cockroachlabs.com/docs/v21.1/operate-cockroachdb-kubernetes#upgrade-the-cluster). Be sure to complete all the steps.
+    1. If you are upgrading to {{ page.version.version }} from a production release earlier than {{ previous_version }}, or from a testing release (alpha/beta), first [upgrade to a production release of {{ previous_version }}]({% link {{ previous_version }}/operate-cockroachdb-kubernetes.md %}#upgrade-the-cluster). Be sure to complete all the steps.
```
Comment (taroface) [inline src/current/v23.1/upgrade-cockroachdb-kubernetes.md:39]:
```suggestion
    1. If you are upgrading to {{ page.version.version }} from a production release earlier than {{ previous_version }}, or from a testing release (alpha/beta), first [upgrade to a production release of {{ previous_version }}]({% link {{ previous_version }}/upgrade-cockroachdb-kubernetes.md %}). Be sure to complete all the steps.
```

File: src/current/v23.2/upgrade-cockroachdb-kubernetes.md [hunk 22]
```diff
@@ -32,11 +32,11 @@ If you [deployed CockroachDB on Red Hat OpenShift]({% link {{ page.version.versi
...
+    To upgrade to a new major version, you must first be on a production release of the previous version. The release does not need to be the latest production release of the previous version, but it must be a production [release]({% link releases/index.md %}) and not a testing release (alpha/beta).
 
     Therefore, in order to upgrade to {{ page.version.version }}, you must be on a production release of {{ previous_version }}.
 
-    1. If you are upgrading to {{ page.version.version }} from a production release earlier than {{ previous_version }}, or from a testing release (alpha/beta), first [upgrade to a production release of {{ previous_version }}](https://www.cockroachlabs.com/docs/v21.1/operate-cockroachdb-kubernetes#upgrade-the-cluster). Be sure to complete all the steps.
+    1. If you are upgrading to {{ page.version.version }} from a production release earlier than {{ previous_version }}, or from a testing release (alpha/beta), first [upgrade to a production release of {{ previous_version }}]({% link {{ previous_version }}/operate-cockroachdb-kubernetes.md %}#upgrade-the-cluster). Be sure to complete all the steps.
```
Comment (taroface) [inline src/current/v23.2/upgrade-cockroachdb-kubernetes.md:39]:
```suggestion
    1. If you are upgrading to {{ page.version.version }} from a production release earlier than {{ previous_version }}, or from a testing release (alpha/beta), first [upgrade to a production release of {{ previous_version }}]({% link {{ previous_version }}/upgrade-cockroachdb-kubernetes.md %}). Be sure to complete all the steps.
```

File: src/current/v24.1/upgrade-cockroachdb-kubernetes.md [hunk 23]
```diff
@@ -32,11 +32,11 @@ If you [deployed CockroachDB on Red Hat OpenShift]({% link {{ page.version.versi
...
+    To upgrade to a new major version, you must first be on a production release of the previous version. The release does not need to be the latest production release of the previous version, but it must be a production [release]({% link releases/index.md %}) and not a testing release (alpha/beta).
 
     Therefore, in order to upgrade to {{ page.version.version }}, you must be on a production release of {{ previous_version }}.
 
-    1. If you are upgrading to {{ page.version.version }} from a production release earlier than {{ previous_version }}, or from a testing release (alpha/beta), first [upgrade to a production release of {{ previous_version }}](https://www.cockroachlabs.com/docs/v21.1/operate-cockroachdb-kubernetes#upgrade-the-cluster). Be sure to complete all the steps.
+    1. If you are upgrading to {{ page.version.version }} from a production release earlier than {{ previous_version }}, or from a testing release (alpha/beta), first [upgrade to a production release of {{ previous_version }}]({% link {{ previous_version }}/operate-cockroachdb-kubernetes.md %}#upgrade-the-cluster). Be sure to complete all the steps.
```
Comment (taroface) [inline src/current/v24.1/upgrade-cockroachdb-kubernetes.md:39]:
```suggestion
    1. If you are upgrading to {{ page.version.version }} from a production release earlier than {{ previous_version }}, or from a testing release (alpha/beta), first [upgrade to a production release of {{ previous_version }}]({% link {{ previous_version }}/upgrade-cockroachdb-kubernetes.md %}). Be sure to complete all the steps.
```

File: src/current/v24.2/upgrade-cockroachdb-kubernetes.md [hunk 24]
```diff
@@ -32,11 +32,11 @@ If you [deployed CockroachDB on Red Hat OpenShift]({% link {{ page.version.versi
...
+    To upgrade to a new major version, you must first be on a production release of the previous version. The release does not need to be the latest production release of the previous version, but it must be a production [release]({% link releases/index.md %}) and not a testing release (alpha/beta).
 
     Therefore, in order to upgrade to {{ page.version.version }}, you must be on a production release of {{ previous_version }}.
 
-    1. If you are upgrading to {{ page.version.version }} from a production release earlier than {{ previous_version }}, or from a testing release (alpha/beta), first [upgrade to a production release of {{ previous_version }}](https://www.cockroachlabs.com/docs/v21.1/operate-cockroachdb-kubernetes#upgrade-the-cluster). Be sure to complete all the steps.
+    1. If you are upgrading to {{ page.version.version }} from a production release earlier than {{ previous_version }}, or from a testing release (alpha/beta), first [upgrade to a production release of {{ previous_version }}]({% link {{ previous_version }}/operate-cockroachdb-kubernetes.md %}#upgrade-the-cluster). Be sure to complete all the steps.
```
Comment (taroface) [inline src/current/v24.2/upgrade-cockroachdb-kubernetes.md:39]:
```suggestion
    1. If you are upgrading to {{ page.version.version }} from a production release earlier than {{ previous_version }}, or from a testing release (alpha/beta), first [upgrade to a production release of {{ previous_version }}]({% link {{ previous_version }}/upgrade-cockroachdb-kubernetes.md %}). Be sure to complete all the steps.
```

**Review by taroface — COMMENTED** [review]
Some suggestions that should fix the broken links.

File: src/current/v23.1/upgrade-cockroachdb-kubernetes.md (hunk 21 above)
Comment (rmloveland) [reply src/current/v23.1/upgrade-cockroachdb-kubernetes.md:39]: Fixed

File: src/current/v23.2/upgrade-cockroachdb-kubernetes.md (hunk 22 above)
Comment (rmloveland) [reply src/current/v23.2/upgrade-cockroachdb-kubernetes.md:39]: Fixed

File: src/current/v24.1/upgrade-cockroachdb-kubernetes.md (hunk 23 above)
Comment (rmloveland) [reply src/current/v24.1/upgrade-cockroachdb-kubernetes.md:39]: updated!

File: src/current/v24.2/upgrade-cockroachdb-kubernetes.md (hunk 24 above)
Comment (rmloveland) [reply src/current/v24.2/upgrade-cockroachdb-kubernetes.md:39]: Fixed!

**Review by taroface — APPROVED** [review]
LGTM!!

**Pull request merged**